### DISCUSSION TITLE
Google Play feedback problem, no one handle, do not automatically translate the language webpage, PAD tablet label display is problematic, overlap, please do not force the computer mode. You can choose the in the hand mode. Do not force adaptation. Otherwise

### DISCUSSION THREAD

**Discussion by zhiduopc** [community/community#34379] [Mobile]
Google Play feedback problem, no one handle, do not automatically translate the language webpage, PAD tablet label display is problematic, overlap, please do not force the computer mode. You can choose the in the hand mode. Do not force adaptation. Otherwise, it will affect the normal use, and it is not convenient to switch labels.
Pad No list of label mode is inconvenient. The mobile side uses the computer label mode. I don't know what the design is?It is very inconvenient to switch one more tags. I hope to improve the list label mode and learn the label mode of Firefox
Hope to take and update as soon as possible, very much looking forward to a lot of people in the feedback of similar issues, if you can support edge, Google Chrome, Firefox browser cross-platform bookmarks, history, tags and other synchronization would be good, will plan faster data migration, replace kiwi use

**Comment by HenSquared** [comment]
Hi! I don't believe I quite understand the feature request or bug here. 

Do you mind letting me know which platform you are using: Android or iOS or browser? 
And 
What is the issue you are facing?

**Reply by zhiduopc** [reply]
Android10，Will not automatically translate foreign language web pages, even if the settings are configured, the default translation page function is invalid, you need to use the proxy to translate web pages, Pad tablet tabs are confused and overlapping, use the computer mode, I hope I can select the list of mobile phone drop-down mode, instead of forcing the computer mode tab, can not select it, automatically adapt to match

**Reply by zhiduopc** [reply]
Sorry, it seems that I sent it to the wrong place, but close it.
